nwbinspector/register_checks.py

```diff
@@ -0,0 +1,97 @@
+"""Authors: Cody Baker and Ben Dichter."""
+from collections import defaultdict, OrderedDict
+from functools import wraps
+
+import h5py
+
+global available_checks
+CRITICAL_IMPORTANCE = 2
+BEST_PRACTICE_VIOLATION = 1
+BEST_PRACTICE_SUGGESTION = 0
+
+importance_levels = OrderedDict(
+    CRITICAL_IMPORTANCE=CRITICAL_IMPORTANCE,
+    BEST_PRACTICE_VIOLATION=BEST_PRACTICE_VIOLATION,
+    BEST_PRACTICE_SUGGESTION=BEST_PRACTICE_SUGGESTION,
+)
```

Comment:
I think it would be simpler and cleaner to use a single Enum class to define the importance levels. In this way the levels are logically grouped together in class, rather than each level being a separate global variable. 

```
from enum import Enum

class Importance(Enum):
   """Importance levels for inspector messages"""
   BEST_PRACTICE_SUGGESTION = 0
   BEST_PRACTICE_VIOLATION = 1
   CRITICAL_IMPORTANCE = 2
   
class Severity(Enum):
  """Severity levels for inspector messages for internal use"""
  LOW_SEVERITY = 4
  HIGH_SEVERITY = 5
```

Reply:
Great find, that class looks perfect for what we're doing!